### PR TITLE
Support SIP DTMF data messages.

### PR DESCRIPTION
**Pull request by dennwc**

### CODE REVIEW

File: pkg/rtc/room.go [hunk 1]
```diff
@@ -1173,27 +1171,27 @@ func (r *Room) sendParticipantUpdates(updates []*participantUpdate) {
...
-		Value: &livekit.DataPacket_Speaker{
-			Speaker: &livekit.ActiveSpeakerUpdate{
-				Speakers: speakers,
-			},
-		},
-	}
+	const kind = livekit.DataPacket_LOSSY
 
 	var dpData []byte
 	for _, p := range r.GetParticipants() {
```
Comment (davidzhao) [inline pkg/rtc/room.go:1177]: we can deprecate this function. all client SDKs supports speaker changed now.

**Review by davidzhao — COMMENTED** [review]
LG! a couple of minor comments.

File: pkg/rtc/participant.go [hunk 2]
```diff
@@ -621,7 +621,7 @@ func (p *ParticipantImpl) OnParticipantUpdate(callback func(types.LocalParticipa
 	p.lock.Unlock()
 }
 
-func (p *ParticipantImpl) OnDataPacket(callback func(types.LocalParticipant, *livekit.DataPacket)) {
+func (p *ParticipantImpl) OnDataPacket(callback func(types.LocalParticipant, livekit.DataPacket_Kind, *livekit.DataPacket)) {
```
[the remark this answers is not in the document]
Comment (dennwc) [reply pkg/rtc/participant.go:624]: It was changed in the PR too: https://github.com/livekit/livekit/pull/2559/files#diff-f3e2dc7cd1a4e17e1a3e505d73ad25a967fcb210f0d3ea4efc9c294a3520f45aR387

File: pkg/rtc/room.go (hunk 1 above)
Comment (dennwc) [reply pkg/rtc/room.go]: Done

**Review by davidzhao — APPROVED** [review]
Awesome! I'm not sure how I missed the interface changes the first time around.

**Review by biglittlebigben — APPROVED** [review]

**Pull request merged**